gnocchi/metric/v1/archivepolicies/results.go

```diff
@@ -0,0 +1,67 @@
+package archivepolicies
+
+import "github.com/gophercloud/gophercloud/pagination"
+
+// ArchivePolicy represents a Gnocchi archive policy.
+// Archive policy is an aggregate storage policy attached to a metric.
+// It determines how long aggregates will be kept in a metric and how they will be aggregated.
+type ArchivePolicy struct {
+	// AggregationMethods is a list of functions used to aggregate
+	// multiple measures into an aggregate.
+	AggregationMethods []string `json:"aggregation_methods"`
+
+	// BackWindow configures number of coarsest periods to keep.
+	// It allows to process measures that are older
+	// than the last timestamp period boundary.
+	BackWindow int `json:"back_window"`
+
+	// Definitions is a list of parameters that configures
+	// archive policy precision and timespan.
+	Definitions []ArchivePolicyDefinition `json:"definition"`
```

Comment:
Let's change this to `Definition` to match the JSON field. It also makes more sense in that this is the "Definition of the ArchivePolicy" rather than "the ArchivePolicy is made up of these Definitions". Make sense?